website/docs/docs/build/incremental-microbatch.md

```diff
@@ -239,6 +240,68 @@ For now, dbt assumes that all values supplied are in UTC:
 
 While we may consider adding support for custom time zones in the future, we also believe that defining these values in UTC makes everyone's lives easier.
 
+## Parallel batch execution
+
+The microbatch strategy offers the benefit of updating a model in smaller, more manageable batches. 
+
+Parallel batch execution means that multiple batches are processed at the same time, instead of one after the other (sequentially). For example, if you have a microbatch model with 12 batches, you can execute those batches in parallel. Specifically they'll run in parallel limited by the number of [available threads](/docs/running-a-dbt-project/using-threads).
+
+To run batches in parallel, use the `concurrent_batches` configuration:
+
+<File name='dbt_project.yml'>
+
+```yaml
+models:
+  +incremental_strategy: "concurrent_batches"
+```
+
+</File>
+
+or:
+
+<File name='models/my_model.sql'>
+
+```sql
+{{
+  config(
+    materialized='incremental',
+    incremental_strategy='concurrent_batches'
+    
+    ...
+  )
+}}
+
+select ...
+```
+
+</File>
+
+It's not possible to run batches concurrently when the model definition contains [`{{ this }}`](/reference/dbt-jinja-functions/this). This is because Jinja represents the state of a table.
```

Comment:
do you think the `{{this}}` should be added in the table too so its not lost? just thinking whether all the caveats need to be in one place so users have the required info in one space/section/place.

Reply:
![Screenshot 2024-12-05 at 09 37 49](https://github.com/user-attachments/assets/929c64ef-4b39-48f9-8215-38e07bd696df)
this table here